pkg/controllers/work/patch_util.go

```diff
@@ -25,17 +26,14 @@ func init() {
 
 // threeWayMergePatch creates a patch by computing a three-way diff based on
 // an object's current state, modified state, and last-applied-state recorded in its annotation.
-func threeWayMergePatch(currentObj, manifestObj client.Object) (client.Patch, error) {
+func threeWayMergePatch(currentObj, manifestObj client.Object, currentObjConfigMap *v1.ConfigMap) (client.Patch, error) {
```

Comment:
Hi Arvind! It might be helpful to take another look at the implementation of the CreateThreeWayMergePatch function; the semantics do not seem to correlate with the args passed in at this moment.

Reply:
Discussed in the meeting, the args match